lib/cli/cmd-init.js

```diff
@@ -19,10 +19,14 @@ module.exports = function(folder, opts) {
       console.log('Initializing new DoneJS application at', folderPath);
       console.log('Installing donejs-cli');
 
-      return installCli(mypkg.version, { cwd: folderPath })
-        .then(function() {
-          return runCliInit(folderPath, opts);
-        });
+      if (!fs.existsSync(path.join(folderModules, 'donejs-cli'))) {
```

Comment:
Is there a reason to not overwrite the cli bin if it already exists? If there is, you'll probably need to write a test for this case.